### PR TITLE
Update DataProviderList.md

### PR DESCRIPTION
**Pull request by mshd**
I would like to add new dataProvider developed by @ogroppo
This DataProvider implementation is optimised for prisma and does not use graphql dependency.

### CODE REVIEW

File: docs/DataProviderList.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ If you can't find a Data Provider for your backend below, no worries! [Writing a
 * **[PostgREST](https://postgrest.org/)**: [raphiniert-com/ra-data-postgrest](https://github.com/raphiniert-com/ra-data-postgrest)
 * **[Prisma](https://github.com/weakky/ra-data-prisma)**: [weakky/ra-data-prisma](https://github.com/weakky/ra-data-prisma)
 * **[Prisma Version 2](https://www.prisma.io/)**: [panter/ra-data-prisma](https://github.com/panter/ra-data-prisma)
+* **[Prisma](https://www.npmjs.com/package/ra-data-simple-prisma)**: [codeledge/ra-prisma](https://github.com/codeledge/ra-prisma)
+
```
Comment (ogroppo) [inline docs/DataProviderList.md:49]: should not have impact on the markdown output but perhaps better not having a new line for consistency

Comment (mshd) [reply docs/DataProviderList.md]: ok

**Comment by djhi** [comment]
What's the difference between your package and the other two already in the list?

**Comment by ogroppo** [comment]
Hi, it does not rely on graphql so the less boilerplate and setup, it's also intended to be nextjs friendly!

**Comment by djhi** [comment]
I see. In this case, would you mind finding a more explicit title so that users know the difference ? Maybe `Prisma REST`? Thanks!

**Comment by mshd** [comment]
@djhi We renamed it to Prisma NextJS because it's optimized for NextJs

**Comment by mshd** [comment]
Any update on the PR @djhi ?

**Comment by fzaninotto** [comment]
I don't see anything related to NextJs in https://github.com/codeledge/ra-prisma-nextjs/tree/main/packages/ra-data-simple-prisma. Perhaps you should underline the absence of dependency on GraphQL instead.

**Comment by mshd** [comment]
> I don't see anything related to NextJs in https://github.com/codeledge/ra-prisma-nextjs/tree/main/packages/ra-data-simple-prisma. Perhaps you should underline the absence of dependency on GraphQL instead.

Ok good point @fzaninotto, it's optimzed for NextJs but obviously isn't restricted to it. I renamed it to Prisma Simple, to highlight the fact that it's a simpler implementation compared to the other packages.

**Comment by fzaninotto** [comment]
I updated the other elements to make the difference clearer

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**